modules/core/package.json

```diff
@@ -47,6 +47,7 @@
     "gen-docs": "typedoc"
   },
   "dependencies": {
+    "@bitgo/tron-lib": "file:../../../bitgo-account-lib",
```

Comment:
This needs to point to an actual, published version of tron lib before this PR can be merged.

Reply:
Will be account-lib before merging